app/controllers/properties_controller.rb

```diff
@@ -0,0 +1,55 @@
+class PropertiesController < ApplicationController
+  def index
+  	@properties = Property.all
```

Comment:
If anything, the index would be a list view of all of the properties for a specific Landlord, so we can make that method or just leave it out since I don't think we will be calling it